server/zanata-war/src/test/java/org/zanata/adapter/GettextAdapterTest.java

```diff
@@ -105,5 +111,48 @@ public void testGettextReference() {
         assertThat(potEntryHeader.getReferences().get(0)).isEqualTo("reference");
     }
 
+    @Test
+    public void testTranslatedGettext() {
+
+        TranslationsResource transResource = new TranslationsResource();
+        addTranslation(transResource, "0293301ed6a54b7e4503e74bba17bf11", "Carpeta padre", ContentState.Approved);
+        addTranslation(transResource, "47a0be8d1015d526a1fbaa56c3102135", "Asunto:", ContentState.Translated);
+        addTranslation(transResource, "49ab28040dfa07f53544970c6d147e1e", "Conectar", ContentState.NeedReview);
+
+        Resource resource = parseTestFile("test-gettext-translated.po");
+        File originalFile = new File(filePath.concat("test-gettext-translated.po"));
+        OutputStream outputStream = new ByteArrayOutputStream();
+
+        adapter.writeTranslatedFile(outputStream,originalFile.toURI(),resource,transResource,"es",Optional.absent());
+
+        assertThat(outputStream.toString()).isEqualTo(
+                "#, fuzzy\n" +
+                        "msgid \"\"\n" +
+                        "msgstr \"\"\n" +
+                        "\"Content-Type: text/plain; charset=UTF-8\\n\"\n" +
+                        "\"Content-Transfer-Encoding: 8bit\\n\"\n" +
+                        "\"MIME-Version: 1.0\\n\"\n\n" +
+                        "msgctxt \"0293301ed6a54b7e4503e74bba17bf11\"\n" +
+                        "msgid \"Parent Folder\"\n" +
+                        "msgstr \"Carpeta padre\"\n\n" +
+                        "msgctxt \"47a0be8d1015d526a1fbaa56c3102135\"\n" +
+                        "msgid \"Subject:\"\n" +
+                        "msgstr \"Asunto:\"\n\n" +
+                        "#, fuzzy\n" +
+                        "msgctxt \"49ab28040dfa07f53544970c6d147e1e\"\n" +
```

Comment:
I think this will be too fragile, because the test would break if we change the msgctxt generation, or the formatting, or eliminate the fuzzy flag from the empty `msgid ""` at the top.

It might be better to (a) parse the file again with JGettext or (b) simply check that the stream contains certain substrings, eg this one:

                        "msgid \"Parent Folder\"\n" +
                        "msgstr \"Carpeta padre\"\n"

What do you think @djansen-redhat ?